docs/make.md

```diff
@@ -30,6 +30,10 @@ Building
 Image building
   build-podman          Build container image using Podman
 
+Dashboard
+  generate-dashboard    Generate dashboard
+  validate-dashboard    Compare dashboard configmaps with git
+
```

Comment:
Is that supposed to be in this PR?

Reply:
No: https://github.com/RHEnVision/provisioning-backend/pull/490